### PR TITLE
[wip, see https://github.com/Talend/components/pull/1083] fix(BAB-58) adding japanese i18n translations

### PR DESCRIPTION
**Pull request by rmannibucau**
**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**Please check if the PR fulfills these requirements**

- [ ] The commit message follows Talend standard
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The code coverage on new code >75%
- [ ] The new code does not introduce new technical issues (sonar / eslint)

**What kind of change does this PR introduce?**

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build / CI related changes
- [ ] Other... Please describe:

**Does this PR introduce a breaking change?**

- [ ] Yes
- [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/1505/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/1505/)

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/1507/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/1507/)

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/1508/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/1508/)

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/1510/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/1510/)

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/1511/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/1511/)

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/1512/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/1512/)

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/1513/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/1513/)

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/1514/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/1514/)

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/1515/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/1515/)

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/1516/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/1516/)

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/1517/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/1517/)

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/1520/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/1520/)

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/1521/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/1521/)

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/1522/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/1522/)

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/1524/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/1524/)

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/1523/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/1523/)

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/1525/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/1525/)

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/1526/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/1526/)

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/1527/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/1527/)

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/1528/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/1528/)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/1529/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/1529/) 
[Sonar results](https://sonar.datapwn.com/overview?id=org.talend.components%3Acomponents%3Armannibucau%2Fja_translations)

File: components/components-jdbc/components-jdbc-definition/src/main/resources/org/talend/components/jdbc/dataset/messages.properties [hunk 1]
```diff
@@ -1,8 +1,8 @@
-definition.JDBCDataset.displayName=JDBC Dataset
-form.Main.title=Database dataset
-form.Main.displayName=Database dataset
-form.CitizenUser.title=Database dataset
-form.CitizenUser.displayName=Database dataset
+definition.JDBCDataset.displayName=JDBC Data Set
+form.Main.title=Database data set
```
Comment (pteyssier) [inline components/components-jdbc/components-jdbc-definition/src/main/resources/org/talend/components/jdbc/dataset/messages.properties]: I remember someone from TFD fix the formulation everywhere to not be a space. He should have fixed Babili too.

File: components/components-jira/src/main/resources/org/talend/components/jira/runtime/writer/JiraWriter.properties [hunk 2]
```diff
@@ -14,5 +14,4 @@ error.notFoundDelete=record wasn''t deleted, because it doesn't exist
 error.invalidRecordUpdate=record update failed
 error.unauthorizedUpdate=user is not authenticated. Record wasn''t updated
 error.forbiddenUpdate=user has no permission to update record
-error.notFoundUpdate=record wasn''t updated, because it doesn''t exist
-error.jsonEmpty=Json content is empty
```
Comment (pteyssier) [inline components/components-jira/src/main/resources/org/talend/components/jira/runtime/writer/JiraWriter.properties]: This key disappeared

**Comment by pteyssier** [comment]
For me, a babili export (or import, question of perspective here) must never modify the basic properties.

File: components/components-jdbc/components-jdbc-definition/src/main/resources/org/talend/components/jdbc/datastore/messages.properties [hunk 3]
```diff
@@ -1,7 +1,7 @@
-definition.JDBCDatastore.title=JDBC Connection
+definition.JDBCDatastore.title=JDBC Data Store
```
Comment (bchen-talend) [inline components/components-jdbc/components-jdbc-definition/src/main/resources/org/talend/components/jdbc/datastore/messages.properties]: see comment for pubsub

File: components/components-pubsub/pubsub-definition/src/main/resources/org/talend/components/pubsub/PubSubDatastoreProperties.properties [hunk 4]
```diff
@@ -1,4 +1,4 @@
-form.Main.title=Pub/Sub Connection Properties
-form.Main.displayName=Pub/Sub Connection Properties
+form.Main.title=Pub/Sub Datastore Properties
```
Comment (bchen-talend) [inline components/components-pubsub/pubsub-definition/src/main/resources/org/talend/components/pubsub/PubSubDatastoreProperties.properties]: idem

File: components/components-kafka/kafka-definition/src/main/resources/org/talend/components/kafka/datastore/KafkaDatastoreProperties.properties [hunk 5]
```diff
@@ -3,5 +3,5 @@ property.brokers.displayName=Bootstrap servers
 properties.ssl.displayName=Use SSL/TLS
 property.possiblevalue.V_0_10_1_0.displayName=0.10.1.0
 property.possiblevalue.V_0_9_0_1.displayName=0.9.0.1
-form.Main.title=Kafka Connection Properties
-form.Main.displayName=Kafka Connection Properties
+form.Main.title=Kafka Datastore Properties
+form.Main.displayName=Kafka Datastore Properties
```
Comment (bchen-talend) [inline components/components-kafka/kafka-definition/src/main/resources/org/talend/components/kafka/datastore/KafkaDatastoreProperties.properties]: idem

File: components/components-jms/jms-definition/src/main/resources/org/talend/components/jms/JmsDatastoreProperties.properties [hunk 6]
```diff
@@ -9,5 +9,5 @@ property.useHttps.displayName=Use Https
 property.httpsSettings.displayName=Https settings
 property.property.displayName=Property
 property.value.displayName=Value
-form.Main.title=Jms Connection Properties
-form.Main.displayName=Jms Connection Properties
+form.Main.title=Jms Datastore Properties
+form.Main.displayName=Jms Datastore Properties
```
Comment (bchen-talend) [inline components/components-jms/jms-definition/src/main/resources/org/talend/components/jms/JmsDatastoreProperties.properties]: idem

File: components/components-fileio/simplefileio-definition/src/main/resources/org/talend/components/simplefileio/s3/S3DatastoreProperties.properties [hunk 7]
```diff
@@ -1,5 +1,5 @@
-form.Main.title=S3 Connection Properties
-form.Main.displayName=S3 Connection Properties
+form.Main.title=S3 Dataset Properties
+form.Main.displayName=S3 Dataset Properties
```
Comment (bchen-talend) [inline components/components-fileio/simplefileio-definition/src/main/resources/org/talend/components/simplefileio/s3/S3DatastoreProperties.properties]: idem

File: components/components-fileio/simplefileio-definition/src/main/resources/org/talend/components/simplefileio/SimpleFileIODatastoreProperties.properties [hunk 8]
```diff
@@ -1,5 +1,5 @@
-form.Main.title=HDFS Connection Properties
-form.Main.displayName=HDFS Connection Properties
+form.Main.title=HDFS Dataset Properties
+form.Main.displayName=HDFS Dataset Properties
```
Comment (bchen-talend) [inline components/components-fileio/simplefileio-definition/src/main/resources/org/talend/components/simplefileio/SimpleFileIODatastoreProperties.properties]: idem

File: components/components-elasticsearch/elasticsearch-definition/src/main/resources/org/talend/components/elasticsearch/ElasticsearchDatastoreProperties.properties [hunk 9]
```diff
@@ -1,5 +1,5 @@
-form.Main.title=Elasticsearch Connection Properties
-form.Main.displayName=Elasticsearch Connection Properties
+form.Main.title=Elasticsearch Datastore Properties
+form.Main.displayName=Elasticsearch Datastore Properties
```
Comment (bchen-talend) [inline components/components-elasticsearch/elasticsearch-definition/src/main/resources/org/talend/components/elasticsearch/ElasticsearchDatastoreProperties.properties]: idem

File: components/components-bigquery/bigquery-definition/src/main/resources/org/talend/components/bigquery/BigQueryDatastoreProperties.properties [hunk 10]
```diff
@@ -1,5 +1,5 @@
-form.Main.title=BigQuery Connection Properties
-form.Main.displayName=BigQuery Connection Properties
+form.Main.title=BigQuery Datastore Properties
+form.Main.displayName=BigQuery Datastore Properties
```
Comment (bchen-talend) [inline components/components-bigquery/bigquery-definition/src/main/resources/org/talend/components/bigquery/BigQueryDatastoreProperties.properties]: it break the change by @pradasouvanlasy , which for datastream, on purpose?
https://github.com/Talend/components/commit/bf42567bb90784b4193fe2c299145a65c585008b

**Comment by rmannibucau** [comment]
superseeded by #1083

**Pull request closed without merging**